src/data/content/community.yaml

```diff
@@ -321,6 +321,27 @@ components:
 
       #CARD MEDIUM
 
+      
+      - title: "Design system .italia cresce grazie al contributo della community"
+        headingLevel: 4
+        customCol: col-lg-4
+        img: /images/community/articoli-di-approfondimento/20240912-contributo-della-community.jpg
```

Comment:
con l'estensione in .jpg non carica l'immagine. Ho già fatto l'upload immagine in .png, bisogna solo cambiare l'estensione qui nel codice prima di approvare la pull request e pubblicare.